portal-bridge/src/bridge/era1.rs

```diff
@@ -368,7 +368,7 @@ impl Era1Bridge {
             let header_hash = block_tuple.header.header.hash();
             let header_content_key = HistoryContentKey::new_block_header_by_hash(header_hash);
             let header_content_info = portal_client.get_content(header_content_key.clone()).await;
-            if let Ok(ContentInfo::Content { .. }) = header_content_info {
+            if let Ok(GetContentInfo { .. }) = header_content_info {
```

Comment:
nit: this could also be `header_content_info.is_ok()`.

Also in 3 more places in this file below.